ff/src/fields/models/quadratic_extension.rs

```diff
@@ -570,7 +570,6 @@ impl<P: QuadExtConfig> From<bool> for QuadExtField<P> {
 impl<P: QuadExtConfig> Neg for QuadExtField<P> {
     type Output = Self;
     #[inline]
-    #[must_use]
```

Comment:
Why do we need to remove it from here as well?

Reply:
Running this from the CI on 1.87.0-nightly, `RUSTFLAGS="-D warnings" cargo check --all-features --examples --workspace --benches` gives:

error: `#[must_use]` has no effect when applied to a provided trait method
   --> ff/src/fields/models/quadratic_extension.rs:573:5
    |
573 |     #[must_use]
    |     ^^^^^^^^^^^